README.md

```diff
@@ -14,6 +14,7 @@ The login UI depends on some extra external native libraries, which you will hav
 
 - @react-native-community/datetimepicker v6
 - disklet v0.5
+- react-native-email-link v1.14.5
```

Comment:
I would just say "v1" here, since the main thing is that the major version number matches, not that the app installs the exact version.